matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/crypto/verification/CancelCode.kt

```diff
@@ -28,7 +28,8 @@ enum class CancelCode(val value: String, val humanReadable: String) {
     MismatchedKeys("m.key_mismatch", "Key mismatch"),
     UserError("m.user_error", "User error"),
     MismatchedUser("m.user_mismatch", "User mismatch"),
-    QrCodeInvalid("m.qr_code.invalid", "Invalid QR code")
+    QrCodeInvalid("m.qr_code.invalid", "Invalid QR code"),
+    Accepted("m.accepted", "Verification request accepted by another device")
```

Comment:
Naming should probably reflect it has been accepted on another device, otherwise it's a bit weird

Reply:
SHould be AcceptedByAnotherDevice as per doc 
> was accepted by a different device. The device receiving this error can ignore the verification request.